### ISSUE TITLE
Incomplete CSV data.

### ISSUE THREAD

**Issue by techspawn** [alexcavender/Automatic-Export-to-CSV-for-Gravity-Forms#26]
Hello alexcavender,

I am exporting data of form and each time I am getting only 15 rows of data. Please help in exporting full data.

Thanks.

**Comment by robgam1234** [comment]
Yes me too, it does it the first time good then after that it doesnt work well.

**Comment by twenty7two** [comment]
I'm also experiencing this bug. Full export on the first try, then only the first 15 entries in the database thereafter. Any idea what this could be?

**Comment by chrizoo** [comment]
same here with new version
